scripts/gen_offset_header.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
```

Comment:
we have standardized on Python 3 for all Zephyr python code, please convert and change this line to 
#!/usr/bin/env python3